catalog-info.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: sda-spring-boot-commons
+  annotations:
```

Comment:
You could include [the docs](https://github.com/SDA-SE/sda-spring-boot-commons/blob/master/mkdocs.yaml) as well.

Reply:
Done! 👍